France-ioi/AlgoreaFrontend
# Feature/split personal data from platform settings

## Description

Fixes #997

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

I've improved "You cannot access this page for this user" message to bring it to standard style

## Test cases

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/split-personal-data-from-platform-settings/en/#/groups/users/670968966872011405/settings)
  3. The I see Platform settings section
  4. Change language to FR
  5. Then I see UI text in FR

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/split-personal-data-from-platform-settings/en/#/groups/users/670968966872011405/personal-data)
  3. The I see only personal data sections
  4. Click on Modify button and update info of user
  5. Then I see updated personal data

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/split-personal-data-from-platform-settings/en/#/groups/users/752024252804317630/personal-data)
  3. The I see "You cannot access this page for this user"

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/split-personal-data-from-platform-settings/en/#/groups/users/752024252804317630/settings)
  3. The I see "You cannot access this page for this user"
